packages/nice-fs/src/asyncMethods.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import fs from 'graceful-fs';
+import util from 'util';
+
```

Comment:
Add comment about `graceful-fs` not supporting `fs.promises` API, which is why we are promisyfing callbacks.